docs/developer-guide/layout-providers/npm-layout-provider.md

```diff
@@ -29,7 +29,9 @@ Following [CLI commands](https://docs.npmjs.com/cli/npm) supported:
 
 - `npm view` 
 
-- `npm search` 
+- `npm search`
+
+- `npm adduser` 
```

Comment:
If you've tested it with both of these, perhaps you could change the line to this:

```suggestion
- `npm [adduser|login]` 
```

...?

Reply:
Updated the text